simplecpp.cpp

```diff
@@ -1716,66 +1716,123 @@ namespace simplecpp {
         return ostr.str();
     }
 #else
-#define realFilename(f)  f
+    // Boring implementation, but could help in platform-independent tests:
+    std::string realFilename(const std::string &f)
```

Comment:
Yes, but as I wrote one might write tests only if there is a real function, cannot write tests for the macro.